requirements.txt

```diff
@@ -1,11 +1,11 @@
 envbash==1.2.0
+# The following packages should ideally be installed through apt
+# For the moment, I am pinning their versions here to the ones available in debian bullseye
```

Comment:
Why are you bumping these versions? I pinned them to those versions to be compatible with #516